.github/workflows/main-pipeline.yml

```
@@ -23,40 +24,42 @@ jobs:
-      run_frontend_audit: ${{ needs.changes-filter.outputs.webapp_changed == 'true' }}
+      run_backend_audit: ${{ fromJSON(needs.changes-filter.outputs.backend_changed) }}
+      run_frontend_audit: ${{ fromJSON(needs.changes-filter.outputs.webapp_changed) }}
 
   build:
     needs: [version, test, openapi-validation, audit]
     uses: ./.github/workflows/build.yml
     with:
       version_tag: ${{ needs.version.outputs.version_tag }}
-      backend_changed: ${{ needs.changes-filter.outputs.backend_changed == 'true' }}
-      webapp_changed: ${{ needs.changes-filter.outputs.webapp_changed == 'true' }}
+      backend_changed: ${{ fromJSON(needs.changes-filter.outputs.backend_changed) }}
```

actionlint (1.7.8)

property "changes-filter" is not defined in object type {audit: {outputs: {}; result: string}; openapi-validation: {outputs: {}; result: string}; test: {outputs: {}; result: string}; version: {outputs: {authors: string; previous_version: string; version_tag: string; version_type: string}; result: string}}

(expression)